pkgs/by-name/gr/grafana-alloy/package.nix

```diff
@@ -0,0 +1,113 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchYarnDeps
+, buildGoModule
+, systemd
+, yarn
+, fixup-yarn-lock
```

Comment:
```
❯ nix-build -A grafana-alloy
error:
       … while calling the 'abort' builtin

         at /home/flokli/dev/nixos/nixpkgs/lib/customisation.nix:269:13:

          268|        # which is especially relevant with allowAliases = false
          269|        else abort "lib.customisation.callPackageWith: ${error}";
             |             ^
          270|

       error: evaluation aborted with the following error message: 'lib.customisation.callPackageWith: Function called without required argument "fixup-yarn-lock" at /home/flokli/dev/nixos/nixpkgs/pkgs/by-name/gr/grafana-alloy/package.nix:8, did you mean "fixup_yarn_lock"?'
```

Reply:
Yeah I ran into this error too, seems to be a very recent change: https://github.com/NixOS/nixpkgs/pull/281902

Reply:
So I assume you need to rebase this PR and update to the new reference?

Reply:
No `fixup-yarn-lock` is the new name. It builds for me on an up-to-date nixpkgs checkout.

Reply:
I see, I cherry-picked it into my nixpkgs' tree and might not have had that rename. It does build now, thanks!